src/main/java/emissary/server/api/Agents.java

```diff
@@ -41,6 +45,18 @@ public Response agents() {
         return Response.ok().entity(lookupAgents()).build();
     }
 
+    @GET
+    @Path("/agents/log")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response agentsLog() {
+        AgentsResponseEntity entity = lookupAgents();
+        AgentsFormatter formatter = AgentsFormatter.builder().withHost(entity.getLocal().getHost()).build();
+        StringJoiner joiner = new StringJoiner("\n", "", "\n");
+        entity.getLocal().getAgents().forEach(agent -> joiner.add("{\"agents\":" + formatter.json(agent) + "}"));
```

Comment:
```suggestion
        entity.getLocal().getAgents().forEach(agent -> joiner.add("{\"agent\":" + formatter.json(agent) + "}"));
```
The generated output is a list like:
```
{"agents": {"timestamp":..."host":..."name":..."status":...}}
{"agents": {"timestamp":..."host":..."name":..."status":...}} 
{"agents": {"timestamp":..."host":..."name":..."status":...}} 
```
the "agents" structure intuitively to me would expect:
```
{"agents":
{"agent": {"timestamp":..."host":..."name":..."status":...}}
{"agent": {"timestamp":..."host":..."name":..."status":...}} 
{"agent": {"timestamp":..."host":..."name":..."status":...}}
}
```
which i think isn't the "format" we were going for